packages/generator-helper/src/__tests__/generatorHandler.test.ts

```diff
@@ -73,6 +73,12 @@ describe('generatorHandler', () => {
 
   // TODO: Windows: this test fails with ENOENT even though the .cmd file is there and can be run manually.
   testIf(process.platform !== 'win32')('parsing error', async () => {
+    // It fails often and randomly on GitHub Actions with
+    // Received promise resolved instead of rejected
+    // Resolved to value: undefined
+    // So we retry it with the hope that it a retry would help
+    jest.retryTimes(3)
```

Comment:
This might be ok for the time being, but I don't think solves the flakiness problem at its source. I'm a bit skeptical of the `GeneratorProcess` definition and its use of `cross-spawn`, which might use some improvements in the future